.github/workflows/perform-release.yaml

```diff
@@ -2,20 +2,19 @@ name: "Perform Release"
 on:
   workflow_dispatch:
     inputs:
-       new_version:
-
-         description: "Release Version (no 'v')"
-         required: true
-         type: string
+      new_version:
+        description: "Release Version (no 'v')"
+        required: true
+        type: string
 
 jobs:
   perform-release:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Update Changelog
-        uses: thomaseizinger/keep-a-changelog-new-release@1.3.0
+        uses: thomaseizinger/keep-a-changelog-new-release@v1
```

Comment:
v1 is broken, this needs to be `v1.3.0`. Trust me, I tried `v1`

Reply:
8d9d9f27afe22c2c8d9b785ef90b8bb98a1404ad